### PR TITLE
Add smithy-build.json instructions for dependencies

### PR DESCRIPTION
**Pull request by srchase**
This PR adds instructions for specifying `smithy-build.json` for use with the Smithy Language Server.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,18 @@ This extension and the Smithy Language Server are not yet published. To use this
...
+{
+    {
+    "maven": {
+      "dependencies": ["software.amazon.smithy:smithy-aws-traits:1.19.0"],
+      "repositories": [{ "url": "https://repo1.maven.org/maven2/" }]
+    }
+  }
+}
+```
+* Start authoring your Smithy model. Opening a `*.smithy` file will active the extension.
```
Comment (kstich) [inline README.md:34]: s/active/activate

File: README.md [hunk 2]
```diff
@@ -20,6 +20,18 @@ This extension and the Smithy Language Server are not yet published. To use this
 * Clone the Extension: `git clone https://github.com/awslabs/smithy-vscode.git && cd smithy-vscode`
 * Run npm commands to install:
 `npm install && npm run install-plugin`
+* Open VS Code and add a `smithy-build.json` file to the root of your project, specifying any Maven dependencies used by your model, along with the repositories where they can be located.
```
Comment (kstich) [inline README.md:23]: Cap line length?

**Review by kstich — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (srchase) [reply README.md]: Fixed in 8aeb801.

File: README.md (hunk 2 above)
Comment (srchase) [reply README.md]: Fixed all the long lines in 8aeb801.

**Review by kstich — APPROVED** [review]

**Pull request merged**